src/pages/HomePage.tsx

```diff
@@ -148,20 +225,40 @@ export class HomepageInternal extends React.Component<
             <Col span={18}>
               {this.state.displayedPackages.length > 0 ? (
                 <>
-                  <h2>Recently Updated</h2>
-                  {this.renderPackageCards()}
+                  {/* 
+                    Recently Updated Packages 
+                    Defaults to showing the 10 last updated packages
+                    Can be changed via dropdown box
+                  */}
+                  <h2>Recently Updated</h2> {/* Set the threshold for "Recently Updated in the function" */}
+                  {this.renderRecentlyUpdatedPackages()}
+                  {/* Let user choose how many recently updated packages to show */}
+                  <div className={"line-under-scrolly"}>
+                    <p className={"line-label"}>Number of packages to display:</p>
+                    <select onChange={(e) => this.changeMaxNumberOfRecentlyUpdatedPackages(e)}>
+                      <option value="5">5</option>
+                      <option value="10" selected>10</option>
+                      <option value="20">20</option>
+                    </select>
+                  </div>
 
                   <Divider />
 
+                  {/* 
+                    My Packages
+                    Renders packages created by the current user
+                  */}
                   <h2>My Packages</h2>
-                  {this.renderPackageCards()}
+                  {this.renderMyPackages()}
 
                   <Divider />
                   <h2>All Packages</h2>
-                  <p>TODO</p>
-                  {/* <Table dataSource={this.state.displayedPackages}>
-                    <Column />
-                  </Table> */}
+                  {/*
+                    AllPackagesTable component displays the "All Packages" section as an antd table
+                    packages: IPackage[] - the packages to display
+                    linkDirectory: string - the directory which these packages are stored under, e.g. for /test/zinnia.unchartedterritory, pass "test" as the prop
+                  */}
+                  <AllPackagesTable packages={this.state.displayedPackages} linkDirectory={this.props.context.selectedPackageSet!.slug} />
```

Comment:
Just a small thing with the naming. So I believe the link directory is pretty much the package set name. So I would change that to package set instead